### ISSUE TITLE
Editing ongoing outage breaks maintenance static page (content and styles)

### ISSUE THREAD

**Issue by golenkovm** [catalyst/moodle-auth_outage#174]
Steps to replicate:
- Create an outage with Auto start maintenance mode ticked.
- Wait until the outage is active and the site is in maintenance mode.
- Disable maintenance mode running `php admin/cli/maintenance.php --disable`.
- Edit the active outage (amend time frame or edit title or description).
- Click Save button.
- Confirm that the site is in maintenance mode and maintenance static page is broken.

**Comment by golenkovm** [comment]
It happens because [curl request](https://github.com/catalyst/moodle-auth_outage/blob/6e960e8814e51a1f69bc09c103fc79529b7d122c/classes/local/outagelib.php#L60) is comming from 127.0.0.1 address and beeing blocked by outage. So the page returned by [this request](https://github.com/catalyst/moodle-auth_outage/blob/6e960e8814e51a1f69bc09c103fc79529b7d122c/classes/local/controllers/maintenance_static_page.php#L63) starts with:

```
<!DOCTYPE html>
<!-- Blocked by ip, your ip: 127.0.0.1 --><html dir="ltr" lang="en" xml:lang="en"><head><title>Title</title><link rel="shortcut icon" href="http://localhost/auth/outage/file.php?file=da39a3ee5e6b4b0d3255$
<div class="toast-wrapper mx-auto py-3 fixed-top" role="status" aria-live="polite"></div>
```

And looks like: 

![image](https://user-images.githubusercontent.com/22066290/73586954-ee66f080-4508-11ea-8152-aa7d46748d1f.png)

**Comment by golenkovm** [comment]
Fixed via #175
